README.md

```diff
@@ -372,10 +372,116 @@ A couple of helpful assertions are provided with the `pollTask` functionality:
 * A given `label` can only be used once. If the same `label` is used a second time, an error will be thrown.
 * If nothing has been queued for the given label, calling `pollTaskFor(label)` will trigger an error.
 
+### `registerDisposable`
+
+**tl;dr call `this.registerDisposable(fn)` on any component, route, or service to register a function you want to run when the object is destroyed.**
+
+Use `registerDisposable` as a replacement for explictly unbinding any external bindings. A disposable is a function that disposes of bindings that are outside of Ember's lifecyle. This essentially means you can register a function that you want to run to automatically tear down any bindings when the Ember object is destroyed.
```

Comment:
Can we keep the description focused on generically running some function upon destroy?  This kind of event binding is only one type of thing where this general functionality is useful (another would be websocket subscriptions, yet another might be interacting with web workers, etc).  It seems great to have these sorts of examples, but the actual purpose of `registerDisposable` is to be super generic...

Reply:
Yep, can do.